tests/README.md

```diff
@@ -13,11 +13,12 @@ Note: Not all tests have been converted to tox and/or pytest yet.
 
 ## Running the tests
 
-Before running vdsm tests, activate the environment:
+Before running Vdsm tests, activate the virtual environment (which you need to
+create once, with `make venv`, as described in [../doc/development.md](../doc/development.md)):
 
     source ~/.venv/vdsm/bin/activate
 
-When done, you can deactivate the environment:
+When done with testing, you can deactivate the environment:
```

Comment:
This info was added now to doc/developmenet.md - do we need to repeat it here?

Reply:
I think it's ok to duplicate it as a reminder, because the setup-info in the other document might have been read at a much earlier time, and now the user looks for testing details in this document and might have forgotten about it.

Reply:
OK, lets keep this duplicated for now.